readme.md

```diff
@@ -204,6 +204,18 @@ To publish [scoped packages](https://docs.npmjs.com/misc/scope#publishing-public
 }
 ```
 
+### Publish private, scoped packages in CI/CD
+
+To publish a [private Org-scoped package](https://docs.npmjs.com/creating-and-publishing-an-org-scoped-package#publishing-a-private-org-scoped-package), you need to set the access level to `restricted`. You can do that by adding the following to your `package.json`:
+
+```json
+"publishConfig": {
+	"access": "restricted"
+}
+```
+
+This will prevent a prompt asking about publically publishing the package.
```

Comment:
`publically` => `publicly`

Also, I think this sentence is a bit unnecessary, and can be removed.